packages/apputils/src/thememanager.ts

```diff
@@ -245,13 +256,23 @@ export class ThemeManager {
     return Promise.all([old, themes[theme].load()])
       .then(() => {
         this._current = theme;
-        Private.fitAll(this._host);
-        splash.dispose();
         this._themeChanged.emit({
           name: 'theme',
           oldValue: current,
           newValue: theme
         });
+
+        // Need to force a redraw of the app here to avoid a Chrome rendering
+        // bug that can leave the scrollbars in an invalid state
+        this._host.hide();
```

Comment:
I am ok with this workaround, but let's open a follow up issue so we have a reminder to revisit this (hopefully Chrome will fix it)